Makefile.toml

```diff
@@ -123,7 +123,7 @@ command = "cargo"
 args = [
     "build",
     "-p",
-    "locked-astroport-vault",
+    "locked-astroport-vault@0.3.0",
```

Comment:
why is this needed?

Reply:
because tree now includes two versions of the crate